src/components/editor/document-bar/permissions/permission-modal.tsx

```diff
@@ -52,9 +52,8 @@ export const PermissionModal: React.FC<PermissionsModalProps> = ({ show, onChang
       list
         .map(user => {
           if (user.id !== userId) {
```

Comment:
Actually this needs to be refactored too.
```suggestion
          if (user.id === userId) {
```